README.md

```diff
@@ -1003,6 +1005,72 @@ For Windows:
 ```
 </details>
 
+### Getting Started with Claude Code
+
+<details>
+<summary>Install in Claude Code</summary>
+
+Configure MCP servers in Claude Code through the CLI or in `.mcp.json`
+
+1. Follow the steps above in the **Installation and Setup** section to install `uv` from [Astral](https://docs.astral.sh/uv/getting-started/installation/), install Python, and configure AWS credentials with the required services.
+
+2. **Using Claude Code CLI Commands**
+
+   Claude Code CLI commands to add MCP servers:
+
+   ```bash
+   # Add core AWS services
+   claude mcp add aws-api uvx awslabs.aws-api-mcp-server@latest
+   claude mcp add aws-cdk uvx awslabs.cdk-mcp-server@latest
+   claude mcp add aws-docs uvx awslabs.aws-documentation-mcp-server@latest
+   claude mcp add aws-support uvx awslabs.aws-support-mcp-server@latest
+   claude mcp add aws-pricing uvx awslabs.aws-pricing-mcp-server@latest
+
+   # Add AI/ML and Bedrock services
+   claude mcp add bedrock-kb uvx awslabs.bedrock-kb-retrieval-mcp-server@latest
+   claude mcp add nova-canvas uvx awslabs.nova-canvas-mcp-server@latest
+   claude mcp add synthetic-data uvx awslabs.syntheticdata-mcp-server@latest
+
+   # Add data and analytics services
+   claude mcp add aws-dataprocessing uvx awslabs.aws-dataprocessing-mcp-server@latest
+   claude mcp add aurora-dsql uvx awslabs.aurora-dsql-mcp-server@latest
+   claude mcp add valkey uvx awslabs.valkey-mcp-server@latest
```

Comment:
Would there be another look through these as a list in the main README please? Thank you!